lib/helpers.js

```diff
@@ -22,19 +23,34 @@ module.exports = {
   },
 
   getClientAddress: function (req) {
-    if (req.connection && req.connection.remoteAddress) {
-      return req.connection.remoteAddress
-    }
-
-    if (req.connection && req.connection.socket && req.connection.socket.remoteAddress) {
-      return req.connection.socket.remoteAddress
-    }
-
-    if (req.socket && req.socket.remoteAddress) {
-      return req.socket.remoteAddress
+    var ipHeaders = [ // Priority
+      'Forwarded', // 1
+      'X-Real-IP', // 2
+      'X-Forwarded-For', // 3
+      'Fastly-Client-IP', // 4
+      'CF-Connecting-IP', // 4
+      'X-Cluster-Client-IP', // 5
+      'Z-Forwarded-For', // 5
+      'WL-Proxy-Client-IP', // 5
+      'Proxy-Client-IP' // 5
+    ]
```

Comment:
These headers could be lowercased by default and stored outside of the function so they are not initialized each time and the loop below isn't required to lowercase them on every iteration.